source/assets/stylesheets/docs/_base.sass

```diff
@@ -41,9 +41,6 @@ h1,h2,h3,h4,h5,h6
 .index[class*="ls-theme-"] p a:not([class*="ls-btn"])
   color: #001e45
 
-ul, ol
```

Comment:
Are you sure?
![screen shot 2015-01-23 at 13 24 27](https://cloud.githubusercontent.com/assets/78355/5877014/42f50770-a303-11e4-9ecb-b6f6f6b4d8d5.png)

Test again. Be careful.